src/main/java/org/jenkinsci/remoting/nio/FifoBuffer.java

```diff
@@ -236,8 +235,7 @@ public int readable() {
      * @return Number of bytes we can write to the buffer.
      *         If the buffer is closed, may return the value beyond the limit (JENKINS-37514)
      */
-    @Nonnegative
-    public int writable() {
+    /* @javax.annotation.Nonnegative */ public int writable() {
```

Comment:
And here we should just remove it also. Especially since it pushes the useful stuff so far right.

Reply:
Ditto